ocs_ci/ocs/platforms/factory.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from ocs_ci.framework import config
+from .aws import AWS
+from .vsphere import VSPHERE
+from .platform import PlatfromBase
+
+
+logger = logging.getLogger(__name__)
+
+
+class PlatformNodesFactory:
+    """
+    A factory class to get specific nodes platform object
+
+    """
+    def __init__(self):
+        self.cls_map = {'AWS': AWS, 'vsphere': VSPHERE, 'aws': AWS, 'baremetal': PlatfromBase}
```

Comment:
instead of adidng AWS key twice ( Capital and small ), can we have one convert into lower in line https://github.com/red-hat-storage/ocs-ci/pull/2021/files#diff-1aa6304189efa87ac6d7d7163657e4ceR21